org.osgi.test.assertj.framework/src/main/java/org/osgi/test/assertj/dictionary/DictionaryConditions.java

```diff
@@ -0,0 +1,70 @@
+/*******************************************************************************
+ * Copyright (c) Contributors to the Eclipse Foundation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ *******************************************************************************/
+
+package org.osgi.test.assertj.dictionary;
+
+import java.util.Collections;
+import java.util.Dictionary;
+import java.util.List;
+import java.util.Map;
+import java.util.Map.Entry;
+import java.util.Objects;
+
+import org.assertj.core.api.Condition;
+import org.osgi.framework.Filter;
+import org.osgi.framework.FrameworkUtil;
+import org.osgi.framework.InvalidSyntaxException;
+import org.osgi.test.common.dictionary.Dictionaries;
+
+/**
+ * A Utility-Class thats Provides static methods to create {@link Condition}s
+ * for an {@link Dictionary}
+ */
+public interface DictionaryConditions {
```

Comment:
These aren't general dictionary conditions. They are service property conditions. That is, `Dictionary<String,Object>`.

Reply:
renamed to `ServicePropertiesConditions`